__tests__/__integration__/imperative/__tests__/__integration__/cli/config/import/cli.imperative-test-cli.config.import.integration.test.ts

```diff
@@ -0,0 +1,324 @@
+            expect(mtimeConfig.getTime()).toBe(lastMtimeConfig.getTime());
+            const mtimeSchema = fs.statSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json")).mtime;
+            expect(mtimeSchema.getTime()).toBe(lastMtimeSchema.getTime());
+        });
+
+        describe("from the web", () => {
+
+            it("should fail to import a config from a bad URL", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    localhostUrl + "/test.config.does.not.exist.json", "--user-config false --global-config false"
+                ]);
+
+                expect(response.stdout.toString()).toEqual("");
+                expect(response.stderr.toString()).toContain("Rest API failure with HTTP(S) status");
+                expect(response.status).toEqual(1);
+
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"))).toEqual(false);
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a schema from a bad URL", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    localhostUrl + "/test.config.bad.with.schema.json", "--user-config false --global-config false"
+                ]);
+
+                expect(response.stderr.toString()).toContain("Failed to download schema");
+                expect(response.status).toEqual(0);
+
+                expectFilesAreEqual(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"),
+                    path.join(__dirname, "__resources__", "test.config.bad.with.schema.json"));
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a config that is invalid JSON from a URL", () => {
+                const response = runCliScript(path.join(__dirname, "/__scripts__/import_config.sh"), TEST_ENVIRONMENT.workingDir, [
+                    localhostUrl + "/test.config.bad.json", "--user-config false --global-config false"
+                ]);
+
+                expect(response.stdout.toString()).toEqual("");
+                expect(response.stderr.toString()).toContain("Unexpected end of JSON input");
+                expect(response.status).toEqual(1);
+
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json"))).toEqual(false);
+                expect(fs.existsSync(path.join(TEST_ENVIRONMENT.workingDir, "test", "test.schema.good.json"))).toEqual(false);
+            });
+
+            it("should fail to import a schema if it is defined with a URL", () => {
```

Comment:
Do we refuse to import a schema defined with a URL because VSCode does not find such a schema and thus provides no IntelliSense?